docs/android/v2/advanced-features/hls-timed-metadata.mdx

```diff
@@ -0,0 +1,48 @@
+---
+ title: HLS Timed Metadata
+ nav: 8.05
+---
+
+HLS Timed Metadata feature helps you synchronise certain events with the HLS stream. This can be useful for showing interactive quizzes / product overlays etc.
+
+## Requirements
```

Comment:
is there any requirements for exoplayer version?

Reply:
nope